README.md

```diff
@@ -81,17 +81,10 @@ on:
   check_suite:
     types:
       - completed
-  pull_request:
+  workflow_run:
+    workflows: ['Continuous Integration']
     types:
-      - edited
-      - labeled
-      - opened
-      - ready_for_review
-      - reopened
-      - synchronize
-      - unlabeled
-      - unlocked
-
+      - completed
 jobs:
```

Comment:
You might want to add the if conclusion = successful check to demonstrate that it automatically merges if you don't add this check.

```
# Add any additional filters to indicate when the merge should be done.
if: ${{ github.event.workflow_run.conclusion == 'success' }}
```

Reply:
👍🏻 
Changed in 
bb0cc8e